UNRELEASED.md

```diff
@@ -6,6 +6,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
+- Added `disabled` prop to `Autocomplete` component.
```

Comment:
This is not the case anymore, I might even call it a bug fix. maybe?

- Added the ability to disable the `Autocomplete` by setting the  `Autocomplete.TextField` `disabled` prop true.